NEXT_CHANGELOG.md

```diff
@@ -41,6 +41,14 @@ It defaults to 30 seconds.
 
 By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/2271
 
+## 🐛 Fixes
+### Traces won't cause missing field-stats ([Issue #2277](https://github.com/apollographql/router/issues/2267))
+
+Previously if a request was sampled for tracing it was contributing to metrics correctly. This was a particular problem for users with a high sampling rate.
```

Comment:
"correctly"?

Reply:
Fixed in [88cf26e](https://github.com/apollographql/router/pull/2277/commits/88cf26ee9fd6f70d000005ef0b505ebfd9588c1f)